skycocker/chromebrew
# Enable Testing a PR from its own repo with crew, and other updates.

- Modified `crew update` to check for these environment variable settings:
`CREW_TESTING_REPO`
`CREW_TESTING_BRANCH`
`CREW_TESTING=1`

If these exist, it will use that branch and repo for the `crew update` git pull.

Reversion is just through unsettling any of those variables and running `crew update`

One could also run `CREW_TESTING=1 crew update` to see if new packages in your PR are broken...

- Fix ccache `env.d` file
- Add `faac`,`libcacard`,`spice_gtk`, `spice_protocols`,`usbredir`
- Update `xorg_server`, `xwayland` and try to mitigate conflicts
- Replace `libopus` with `opus` (and adjust dependencies in packages using `opus`) 
- Remove duplicate `libvte` since more recently updated `vte` exists (and adjust dependencies in packages using `vte`) 
- Add `ncursesw` from @saltedcoffii 
- Update `gtk_vnc`, `libsodium`, `libvncserver`,`pipewire',`remmina`, `freerdp`
- Fix `usbutils` curl line in postinstall.
- Fix broken `texlive` package

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686
